discord/interactions.py

```diff
@@ -503,6 +504,44 @@ async def delete_original_response(self) -> None:
             proxy_auth=http.proxy_auth,
         )
 
+    async def translate(
+        self, string: Union[str, locale_str], *, locale: Locale = MISSING, data: Optional[Any] = MISSING
+    ) -> Optional[Any]:
+        """|coro| Translates a string using the set :class:`~discord.app_commands.Translator`.
+
+        .. versionadded:: 2.1
+
+        Parameters
+        ----------
+        string: Union[:class:`str`, :class:`~discord.app_commands.locale_str`]
+            The string to translate.
+            :class:`~discord.app_commands.locale_str` can be used to add more context,
+            information, or any metadata necessary.
+        locale: :class:`Locale`
+            The locale to use, this is handy if you want the
+            translation for a specific locale.
+            Defaults to the user's :attr:`.locale`.
+        data: Optional[Any]
```

Comment:
The rest of documentation does not explicitly documents the type for parameters of `Any` type.
```suggestion
        data:
```